### PR TITLE
RHDEVDOCS-3179 Update release notes for 5.0.6

### PR DESCRIPTION
**Pull request by rolfedh**
- Aligned team: Dev Tools
- For branches: 4.7+
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-3179
- Direct link to doc preview: https://deploy-preview-34814--osdocs.netlify.app/openshift-enterprise/latest/logging/cluster-logging-release-notes.html#cluster-logging-release-notes-5-0-6
- SME review: @vimalk78 
- QE review: @anpingli 
- Peer review: @jc-berger 
- All reviews complete. Please merge now.

### CODE REVIEW

**Review by anpingli — COMMENTED** [review]
/lgtm

File: modules/cluster-logging-release-notes-5.1.0.adoc [hunk 1]
```diff
@@ -80,5 +80,5 @@ If you configure the `ClusterLogging` custom resource (CR) to switch from epheme
 +
 (link:https://issues.redhat.com/browse/LOG-1351[*LOG-1351*])
 
-Before this update, if you redeployed a full Elasticsearch cluster, it got stuck in an unhealthy state, with one non-data node running and all other data nodes shut down. This happened because new certificates prevented the Elasticsearch Operator from scaling down the non-data nodes of the Elasticsearch cluster. With this update, Elasticsearch Operator can scale all the data and non-data nodes down and then back up again, so they load the new certificates. The Elasticsearch Operator can reach the new nodes after they load the new certificates.
+* Before this update, if you redeployed a full Elasticsearch cluster, it got stuck in an unhealthy state, with one non-data node running and all other data nodes shut down. This happened because new certificates prevented the Elasticsearch Operator from scaling down the non-data nodes of the Elasticsearch cluster. With this update, Elasticsearch Operator can scale all the data and non-data nodes down and then back up again, so they load the new certificates. The Elasticsearch Operator can reach the new nodes after they load the new certificates.
```
Comment (jc-berger) [inline modules/cluster-logging-release-notes-5.1.0.adoc:83]: the phrasing `it got stuck in an unhealthy state` sounds a little like a colloquialism. 

Could you rephrase to something like, 

`Before this update, if you redeployed a full Elasticsearch cluster, it would have one non-data node running and all other data nodes shut down. `

Comment (jc-berger) [inline modules/cluster-logging-release-notes-5.1.0.adoc:83]: I try to avoid words like `it, this, there, that' as subjects of my sentences. 

Instead of `This happened because new certifications prevented...` consider saying, `This error happened because...` or `This shut down happened because...` or `This state happened because...`

Comment (rolfedh) [reply modules/cluster-logging-release-notes-5.1.0.adoc:83]: I wondered the same thing, Jake, and checked the documentation. The term, "unhealthy," appears in the UI and many parts of our documentation.

**Review by jc-berger — APPROVED** [review]
/lgtm

**Review by Preeticp — APPROVED** [review]
/lgtm
